### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Legion-81** [CitiesSkylinesMultiplayer/CS2M#5]
Super excited to see a multiplayer mod. Is this what its going to be?

**Comment by Iwo24pl** [comment]
Hey! I don't know what you think it's going to be, but this will be a mod that allows you to play Cities Skylines 2 with friends!

**Comment by Legion-81** [comment]
Yes thats exactly what I thoughts its going to be! Keep up the great work. Thank you for creating this 💯

**Comment by Legion-81** [comment]
> What's the ETA before it starts working?

obviously its done when its done! Just be happy that people who do this are working for FREE on this. Can't wait for this to be done either but don't stress anyone :) When its done i am sure they will announce it.
